modules/nested-tabs-html/assets/js/frontend/handlers/nested-tabs-html.js

```diff
@@ -531,21 +535,21 @@ export default class NestedTabsHtml extends Base {
 	}
 
 	setTouchMode() {
-		if ( elementorFrontend.isEditMode() ) {
+		const widgetSelector = this.getSettings( 'selectors' ).widgetContainer;
+
+		if ( elementorFrontend.isEditMode() || 'resize' === event?.type ) {
 			const responsiveDevices = [ 'mobile', 'mobile_extra', 'tablet', 'tablet_extra' ],
 				currentDevice = elementorFrontend.getCurrentDeviceMode();
 
 			if ( -1 !== responsiveDevices.indexOf( currentDevice ) ) {
-				this.$element.find( '.e-n-tabs' ).attr( 'data-touch-mode', 'true' );
+				this.$element.find( widgetSelector ).attr( 'data-touch-mode', 'true' );
```

Comment:
I am using the same function for the mega menu and therefore I needed dynamic selectors.